### ISSUE TITLE
Front colophon items to add

### ISSUE THREAD

**Issue by mitchkeller** [rbeezer/mathbook#355]
DTD allows `<p>` inside `<colophon>`, and HTML output renders it. However, LaTeX does not. Since there's no ISBN element, I was trying to put the ISBN on my copyright page with `<p>` and just realized that the LaTeX converter is totally ignoring the contents of my `<p>`.

**Comment by rbeezer** [comment]
*  There should be an `ISBN` element, my prototye FCLA had one.
*  I use `colophon` twice, front and back.  The `p` is for back (simple), the rest is for front (laborious).

This is a shortcoming in the DTD that I hope RELAX NG will address.

How about retitling this to ask for front colophon items generally and give me a complete list of what you need (or maybe a good question for the list?).  I need this for my FCLA conversion, which is still in limbo.

Ideas at top of:
https://github.com/rbeezer/fcla/blob/master/src/bookinfo.xml

**Comment by mitchkeller** [comment]
Definitely should have `ISBN`, and I can't find that string in any of the `xsl` files for MBX, so it must have disappeared at some point. `edition` or `version` (I prefer edition, but I saw version in your FCLA link and so I figured I'd throw them both out.)

Eventually, it would be good to be able to support something close to the Library of Congress Cataloging in Publication (CIP) data that's typically part of the front colophon for a commercially-published text, but I'd say getting `ISBN` and `edition` in there for now would be a good resolution.

**Comment by mitchkeller** [comment]
So there is code in `mathbook-latex.xsl` to read an `edition` from `colophon`, but not in DTD. Output to HTML as well, but without any additional text. (LaTeX adds **Edition:**.)

Structure for `website` in `colophon` or `docinfo` could use some work. (DTD calls for it in `docinfo`, but `mathbook-latex.xsl` seems to suggest that it needs to be in `colophon` for it to process.) LaTeX processor expects `address` and `name`. Assuming that part of the point of putting the website in the front colophon is for print editions, using anything but the URL for the `name` seems silly, but then putting in the URL for `name` doesn't get the nice URL treatment that `url` does when you just specify `@href` and leave an empty element. I guess my suggestion would be to let `website` still have the two elements, but for the front colophon, only use the `address` and format like empty `url` elements in text body.

**Comment by rbeezer** [comment]
Thanks, Mitch.  I think much of this is sorted now at 34574c7e.  `website` belongs in `frontmatter/colophon` and I harmonized, implemented, `edition` properly (`version` is still a good idea too).

`website/name` is not a bad idea in print.  Maybe your website is popular enough to go by a brand name (think "Wikipedia").  OK, maybe a long time in coming.  I like to put the domain name and skip the `http://` part.  So default is main font.  If you like a monospace look, then wrap `name` contents in a "c" tag.

I'm going to close this for the `website` and `edition` parts, and I've made #386 to add (not fix) a few more.

Rob
